### DISCUSSION TITLE
Just released my app

### DISCUSSION THREAD

**Discussion by paulorrfilho** [google/filament#4498] [Show and tell]
I finally finished porting my app from iOS to Android. Low-end devices are suffering a bit with lags but reviews are mostly good so far.
If you happen to try it on your phone I would appreciate your feedback.
You can download it [here](https://play.google.com/store/apps/details?id=com.ribeirop.drumknee)

**Comment by romainguy** [comment]
Thanks! Are you using dynamic resolution for low end devices?

**Reply by paulorrfilho** [reply]
The dynamic resolution setting was buggy during the testing phase. I think this was discussed in some thread here. Not sure what to do about it yet, will likely re-visit soon.

**Reply by romainguy** [reply]
Let us know if you try again and run into problem. Another thing you can try if you haven't done so already is to use a lower resolution than native resolution. It's easy to do with Surface/TextureView. Congrats on your release!

**Comment by pixelflinger** [comment]
Congratulations!
